src/components/UserDetailSections/UserLoans/UserLoans.js

```diff
@@ -97,7 +98,10 @@ class UserLoans extends React.Component {
               <li key={index}>
                 <Link
                   id={item.id}
-                  to={`/users/${params.id}/loans/${item.status}`}
+                  to={{
+                    pathname: `/users/${params.id}/loans/${item.status}`,
+                    state: { search: location.search },
```

Comment:
I wondering if `location.state` should be combined with the search value. As I see in most cases you always pass `location.state` as a `state` param upon redirect.

Reply:
If we use a search property, it will be displayed in url. I'm not sure if it's okay to display unused properties everywhere so I preferred to hide search(query) parameters in the state.

Reply:
Fair enough.